Nimble/Expectation.swift

```diff
@@ -1,5 +1,25 @@
 import Foundation
 
+func expectTo<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, to: String) -> (Bool, FailureMessage) {
+    var msg = FailureMessage()
+    msg.to = to
+    let pass = matcher.matches(expression, failureMessage: msg)
+    if msg.actualValue == "" {
+        msg.actualValue = "<\(stringify(expression.evaluate()))>"
+    }
+    return (pass, msg)
+}
+
+func expectToNot<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, toNot: String) -> (Bool, FailureMessage) {
+    var msg = FailureMessage()
+    msg.to = toNot
+    let pass = matcher.doesNotMatch(expression, failureMessage: msg)
+    if msg.actualValue == "" {
+        msg.actualValue = "<\(stringify(expression.evaluate()))>"
+    }
+    return (pass, msg)
+}
```

Comment:
Sounds good. I'm just not sure what of a good name :blush:. They're taken directly from the original `to` and `toNot` methods. It's mostly bootstrapping of the expectation. It's a messy part that I'm not sure the best way to break it apart.